files/en/networking/tr069_acs/tr069_acs.md

```diff
@@ -293,3 +293,80 @@ Now we can see devices for the upgrade (from this group) and some parameters for
 Click on this button to update a device.
 
 ![ready for upgrade](ready_for_update.png)
+
+### Some features description
+
+<details>
+<summary><b>Relation with inventory</b></summary>
+<p markdown="1">
+It is possible  to assign device by ACS identifier (Serial number) to customer before device establishes first connection to ACS. In this way you can add an inventory item with a serial number specified and assign it to a customer:
+
+![item](item.png)
+
+After this you can setup some group and auto provisioning, for example to push PPPoE login&password, wifi SSID&password on first connect. After this with first connect a TR-069 device will be automatically assigned to a customer and initial configuration will be pushed:
+
+![group_mikrotik](group.png)
+
+and after first connect:
+
+![first](device_assigned.png)
+
+</p>
+</details>
+
+<details>
+<summary><b>Device management by customer from portal page</b></summary>
+<p markdown="1">
+The customer can reboot and change SSID and Wi-Fi password for assigned devices directly from portal page on menu Service -> Hardware:
+
+![manage](device_on_portal.png)
+
+</p>
+</details>
+
+
+### Troubleshooting
+
+Follow these instructions to troubleshoot problems with TR-069 (ACS):
+
+<details>
+<summary><b>Can't connect device to ACS</b></summary>
+<p markdown="1">
+
+  0. Make sure that device is accessible from Splynx server. Use tools like ping or traceroute to ensure;
+
+  1. Pay attention to IP restrictions. Requests have to come from IP/network allowed in this list:
+
+  ![ip_restriction](ip_restriction.png)
+
+  2. Make sure that CWMP URL, username and password what are configured under TR-069 client on a device is correct and ports 7547, 7567 are opened;
+
+  3. Disable firewall on a device to ensure that it's not a firewall issue;
+
+  4. In case of using HTTPS - try to disable option "SSL enabled by default", and use CWMP URL like http://splynxserver.com:7547/ instead of https at the beginning.
+
+  5. Restart GenieACS services using this command - "sudo service genieacs* restart"
+
+</p>
+</details>
+
+<details>
+<summary><b>Device is registered in Splynx but no communication</b></summary>
+<p markdown="1">
+  0. Make sure that device is accessible from Splynx server. Use tools like ping or traceroute to ensure;
+
+  1. Pay attention to IP restrictions. Requests have to come from IP/network allowed in this list:
+
+  ![ip_restriction](ip_restriction.png)
+
+  2. Refresh device using this option and wait for the result:
+
+  ![refresh_Device](refresh_Device.png)
+
+  3. Pay attention to windows "Tasks" and "Faults". Try to remove all tasks and all faults using remove button:
```

Comment:
to windows > to the windows; 

remove all tasks and all faults using remove button > all the tasks and the related faults using the remove button;